src/translib/transformer/xfmr_vrf.go

```diff
@@ -231,28 +243,69 @@ var network_instance_table_name_xfmr TableXfmrFunc = func (inParams XfmrParams)
                 return tblList, errors.New("Invalide network instance name")
         }
 
-        /*
-         * for GET or DELETE, use the key "name" only to get the DB table name
-         * for CREATE, UPDATE or REPLACE, use the key "name" and type to get the DB table name
-         */
+        /* For CREATE or PATCH at top level (Network_instances), check the config type if user provides one */
         if ((inParams.oper == CREATE) ||
-            (inParams.oper == UPDATE) ||
-            (inParams.oper == REPLACE))  {
-                /* get the type for the network instance config */
+            ((inParams.oper == UPDATE) && (inParams.requestUri == "/openconfig-network-instance:network-instances"))) {
                 oc_nwInstType, ierr := getNwInstType(nwInstObj, keyName)
                 if (ierr != nil ) {
```

Comment:
If the operation is "UPDATE", can we do DB get irrespective URI is for VRF config/any child config (e.g BGP)?

Reply:
The check  (inParams.requestUri == "/openconfig-network-instance:network-instances") mandates, that the requests are only for network-instances.